content/en/seavoice/discord/discord_bot.md

```diff
@@ -30,13 +30,20 @@ We feel that providing highly accurate transcriptions for voice channels is a hu
 ### ✍️ Speech-to-Text
 ##### Transcribe Audio from Voice Channels
 
-###### /recognize
+###### /recognize [language]
 
-`/recognize` -> Bot joins the voice channel you're currently in, and continues to listen and output transcription in real time to the chat channel. The bot will record and transcribe everyone in the voice channel. Transcriptions are output to the text channel where the initial slash command was entered. When the session ends, the bot will DM each participant a final transcription file and a link to a full audio download.
+`/recognize [language]` -> Bot joins the voice channel you're currently in, and continues to listen and output transcription in real time to the chat channel. The bot will record and transcribe everyone in the voice channel. Transcriptions are output to the text channel where the initial slash command was entered. When the session ends, the bot will DM each participant a final transcription file and a link to a full audio download. The session will automatically wrap up if all the users leave the voice channel, or if the bot shuts down or restarts for any reason (such as when a new verison gets released).
 
-<p style="color:#19b6c0">Pro Tip:</p>
+###### Language Support
 
-Use the `/recognize` command from the voice channel chat window to see your transcriptions side-by-side with the participants or live stream!
+| Language          |
+| ----------------- |
+| English           |
+| Taiwanese Mandarin|
```

Comment:
Shall we call it Mandarin (Taiwan) so that we can sort them by the language?

Reply:
Sure, it's already that way in the actual discord bot, i'll update the docs to be consistent